python/tank/platform/qt/login_dialog.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+try:
+    from . import QtCore, QtGui
+except ImportError:
+    try:
+        from PySide import QtCore, QtGui
+    except ImportError:
+            from PyQt import QtCore, QtGui
+
+from tank.errors import TankAuthenticationError
+
+from . import resources_rc
+from . import ui_login
+from tank.util import authentication
+
+
+class LoginDialog(QtGui.QDialog):
+    """
+    Dialog for getting user crendentials.
+    """
+    def __init__(self, title, session_renewal, parent=None, **kwargs):
```

Comment:
Actually, I don’t either. I meant to change that bit. Thanks for pointing it out.